docs/source/development/reference/functions.rst

```diff
@@ -391,6 +391,11 @@ Coordinate transformation
               reasons.
 
 
+
+.. doxygenfunction:: proj_trans_bounds
```

Comment:
the failure is due to 4D_api.cpp not being set in INPUT of Doxyfile (there might be issues since this will be the first time Doxygen is run onto it)

Reply:
I see the issues in the build. 
```
Warning, treated as error:
PROJ/docs/source/development/reference/functions.rst:395:doxygenfunction: Cannot find function "proj_trans_bounds" in doxygen xml output for project "doxygen_api" from directory: ../build/xml/
make: *** [html] Error 2
Makefile:31: recipe for target 'html' failed
Error: Process completed with exit code 2.
```
Is this something that will fix itself in future runs?